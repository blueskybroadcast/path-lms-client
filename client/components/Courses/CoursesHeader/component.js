import React from 'react';

const CoursesHeader = () => (
  <header
    className="with-banner"
    id="main-container-header"
  >
    <hgroup>
      <h2>
        <i className="icon icon-course" />
        <span>&nbsp;Explore Programs</span>
      </h2>
    </hgroup>
  </header>
);

export default CoursesHeader;
