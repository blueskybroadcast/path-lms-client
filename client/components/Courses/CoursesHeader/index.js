import LibraryHeader from './component';

export default LibraryHeader;
