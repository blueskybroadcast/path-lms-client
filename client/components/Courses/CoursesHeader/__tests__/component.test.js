import React from 'react';
import { shallow } from 'enzyme';
import CoursesHeader from '../component';

describe('Courses header', () => {
  test('should render CoursesHeader correctly', () => {
    const wrapper = shallow(
      <CoursesHeader />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
