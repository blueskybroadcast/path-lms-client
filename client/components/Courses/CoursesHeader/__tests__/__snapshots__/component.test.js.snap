// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Courses header should render CoursesHeader correctly 1`] = `
<header
  className="with-banner"
  id="main-container-header"
>
  <hgroup>
    <h2>
      <i
        className="icon icon-course"
      />
      <span>
        Â Explore Programs
      </span>
    </h2>
  </hgroup>
</header>
`;
